Pandas/pandas_movie.py

```diff
@@ -16,52 +16,49 @@
 movie_data = pd.merge(movie_ratings, users)
 
 # Top rated movies
-print "Top rated movies (overall): \n" , movie_data.groupby('title').size().order(ascending=False)[:20]
-print "\n"
-print "\n"
+print("Top rated movies (overall): \n" , movie_data.groupby('title').size().sort_values(ascending=False)[:20])
+print()
 
 # Find top rated movies for teenagers and old people
 
 oldies = movie_data[(movie_data.age > 60)]
 # Get the top rated ones
-oldies = oldies.groupby('title').size().order(ascending=False)
+oldies = oldies.groupby('title').size().sort_values(ascending=False)
 
 # Extract movies for teens
 teens = movie_data[(movie_data.age > 12) & (movie_data.age < 20)]
 # Get the top rated ones
-teens = teens.groupby('title').size().order(ascending=False)
-
-print "Top ten movies for teens: \n", teens[:10]
-print "\n"
-print "Top ten movies for oldies: \n", oldies[:10]
-print "\n"
-print "\n"
+teens = teens.groupby('title').size().sort_values(ascending=False)
 
+print("Top ten movies for teens: \n", teens[:10])
+print()
```

Comment:
I see what you are doing now. Replace

print()

with

print("\n")

That will do